tests/integration/goerly/debug_traceTransaction/test_1.json

```diff
@@ -1,28 +0,0 @@
-[
```

Comment:
May I ask why are we deleting this integration test?

Reply:
See previous comment (there is some additional mistake to fix)